File: jar/src/main/java/jline/solvers/ba/BA.java
/*
 * Copyright (c) 2012-2026, QORE Lab, Imperial College London
 * All rights reserved.
 */

package jline.solvers.ba;

import jline.lang.Network;
import jline.solvers.SolverOptions;

/**
 * BA is an alias for SolverBA (Bound Analysis solver).
 */
public class BA extends SolverBA {

    public BA(Network model, String method) {
        super(model, method);
    }

    public BA(Network model, SolverOptions options) {
        super(model, options);
    }

    public BA(Network model, Object... varargin) {
        super(model, varargin);
    }

    public BA(Network model) {
        super(model);
    }
}

File: matlab/src/solvers/BA.m
classdef BA < SolverBA
    % BA - Alias for SolverBA (Bound Analysis solver)
    %
    % Copyright (c) 2012-2026, Imperial College London
    % All rights reserved.

    methods
        function self = BA(model, varargin)
            % BA(MODEL, VARARGIN)
            self@SolverBA(model, varargin{:});
        end
    end
end
